projects/igniteui-angular/src/lib/grids/columns/column.component.ts

```diff
@@ -43,10 +43,13 @@ import {
     IgxCollapsibleIndicatorTemplateDirective,
     IgxFilterCellTemplateDirective
 } from './templates.directive';
-import { MRLResizeColumnInfo, MRLColumnSizeInfo } from './interfaces';
+import { MRLResizeColumnInfo, MRLColumnSizeInfo, IColumnPipeArgs } from './interfaces';
 import { DropPosition } from '../moving/moving.service';
 import { IgxColumnGroupComponent } from './column-group.component';
 
+const DEFAULT_DATE_FORMAT = 'mediumDate';
+const DEFAULT_DIGITS_INFO = '1.0-3';
```

Comment:
Could you check with Ignite UI for jQuery and the numeric editor. I think default there was 1.0-2? Also is the Angular pipe default 1.0-3?

Reply:
Yes Koce, the default value is based on the Angular default pipe:

```
{​​​​​​​​minIntegerDigits}​​​​​​​​.{​​​​​​​​minFractionDigits}​​​​​​​​-{​​​​​​​​maxFractionDigits}​​​​​​​​.

minIntegerDigits: The minimum number of integer digits before the decimal point. Default is 1.
minFractionDigits: The minimum number of digits after the decimal point. Default is 0.
maxFractionDigits: The maximum number of digits after the decimal point. Default is 3.

```